### ISSUE TITLE
Readme: Instagram access token generation appears to need "&scope=public"

### ISSUE THREAD

**Issue by korakinos** [keshrath/ImageLoader#2]
When authenticating with Instagram via an "access token" generated as described in the [Readme](https://github.com/keshrath/ImageLoader/blob/64c2efd93d48c3e22bba178d8ea2d10568f171f4/README.md), I get this error on running the [example program](https://github.com/keshrath/ImageLoader/blob/64c2efd93d48c3e22bba178d8ea2d10568f171f4/examples/InstagramLoaderExample/InstagramLoaderExample.pde):

```
 INFO (InstagramLoader) - Starting Thread: Thread[InstagramTask,5,main]...
 INFO (InstagramTask) - Task is running...
ERROR (InstagramTask) - An error occured. The task will be stopped.
org.jinstagram.exceptions.InstagramBadRequestException: OAuthPermissionsException: This request requires scope=public_content, but this access token is not authorized with this scope. The user must re-authorize your application with scope=public_content to be granted this permissions.
    at org.jinstagram.entity.common.InstagramErrorResponse.throwException(InstagramErrorResponse.java:44)
    at org.jinstagram.Instagram.handleInstagramError(Instagram.java:1073)
    at org.jinstagram.Instagram.createInstagramObject(Instagram.java:1037)
    at org.jinstagram.Instagram.getRecentMediaTags(Instagram.java:831)
    at org.jinstagram.Instagram.getRecentMediaTags(Instagram.java:789)
    at at.mukprojects.imageloader.instagram.InstagramTask.run(Unknown Source)
    at java.lang.Thread.run(Thread.java:745)
 INFO (InstagramTask) - Task has stopped.
```

When I re-generate my access token with "&scope=public" appended to the URL like this:
`https://instagram.com/oauth/authorize/?client_id=[CLIENT_ID_HERE]&redirect_uri=http://localhost&response_type=token&scope=public`
… the error disappears.

**Comment by keshrath** [comment]
Hi,
Thanks for the feedback. You are right, the scope needs to be public. I guess your first your first issue is resolved as well by this change?

**Comment by korakinos** [comment]
Hi, thanks!

Howerver, the [other issue](https://github.com/keshrath/ImageLoader/issues/1) about InstagramLoader not loading images remains unresolved. (I just checked again to be sure.) It is not very urgent for me, though, since I realised that I need to query for a specific username or tag, which ImageLoader doesn't seem to implement (only general search, or am I missing something?); so I need to look for another solution anyway.
